utils/utils.py

```diff
@@ -220,14 +220,18 @@ def append_df_to_excel(filename, df, sheet_name='Sheet1', startrow=None,
     # save the workbook
     writer.save()
 
-def save_config(json_opts, json_filename, model, val_loss_log, best_loss):
-    """
-    function that saves the experiment results and tests the model
-    """
-
+def save_config(json_opts, json_filename, model):
+    '''
+    Save experiment config and model path
+    :param json_opts: experiment config object
+    :param json_filename: experiment config filename
+    :param model: current model
+    :return: path_to_model
+    '''
     # get the model path with the epoch of the best model
+    # Todo - model path is not yet accurate
     model_path = os.path.join(model.save_dir, '{0:03d}_net_{1}.pth'.format(
-        val_loss_log.loc[val_loss_log['Seg_Loss'] == best_loss, 'epoch'].item(),
+        model.best_epoch,
         json_opts.model.model_type))
```

Comment:
@Jimmy2027 : How do you name models upon saving and when do you save them to find them like this here?

Reply:
my save function:
```
def save(self, network_label, epoch_label):
       self.save_network(self.net, network_label, epoch_label, self.gpu_ids)
```
I save the weights of the model every epoch, it takes about 0.07077169418334961 secs.